controllers/profiles/dev/states_dev.go

```diff
@@ -70,15 +73,14 @@ func (e *ensureRunningWorkflowState) Do(ctx context.Context, workflow *operatora
 
 	devBaseContainerImage := workflowdef.GetDefaultWorkflowDevModeImageTag()
 	// check if the Platform available
-	pl, err := platform.GetActivePlatform(ctx, e.C, workflow.Namespace)
-	if err == nil && len(pl.Spec.DevMode.BaseImage) > 0 {
-		devBaseContainerImage = pl.Spec.DevMode.BaseImage
+	if plf != nil && len(plf.Spec.DevMode.BaseImage) > 0 {
+		devBaseContainerImage = plf.Spec.DevMode.BaseImage
 	}
 	userPropsCM, _, err := e.ensurers.userPropsConfigMap.Ensure(ctx, workflow)
 	if err != nil {
 		return ctrl.Result{Requeue: false}, objs, err
 	}
-	managedPropsCM, _, err := e.ensurers.managedPropsConfigMap.Ensure(ctx, workflow, pl, common.ManagedPropertiesMutateVisitor(ctx, e.StateSupport.Catalog, workflow, pl, userPropsCM.(*corev1.ConfigMap)))
+	managedPropsCM, _, err := e.ensurers.managedPropsConfigMap.Ensure(ctx, workflow, plf, common.ManagedPropertiesMutateVisitor(ctx, e.StateSupport.Catalog, workflow, plf, userPropsCM.(*corev1.ConfigMap)))
```

Comment:
Why have you changed to `pl` in some cases and here you use `plf`? 😄

Reply:
Honestly, I don't know, perhaps it was during one of my re-re-re-factorings... I intended to change all variables named `platform` to avoid collision with the one package with that name, maybe I went too far here because there was no reason for the change 😄. Do you want me to revert it?

Reply:
No, don't need to bother unless you need to make another change in this PR.